media-domain/src/main/java/nl/vpro/domain/media/MediaObjects.java

```diff
@@ -1060,9 +1066,16 @@ public static boolean willBePlayable(@NonNull Platform platform, @NonNull MediaO
         AVFileFormat.MP3,
         AVFileFormat.MP4,
         AVFileFormat.M4V,
-        AVFileFormat.H264
+        AVFileFormat.H264,
+        AVFileFormat.HASP
```

Comment:
dit klopt alleen voor npo en npo+drm scheme. in andere situaties is dit webonly content en die is niet afspeelbaar.

dus ik zou die hasp check eruit halen

Reply:
ok.